### PR TITLE
Browser manager support

### PR DESCRIPTION
**Pull request by quigamdev**
Selenium.WebDriver is distributed with browser manager since version 4.11. Browser manager makes sure that driver and browser are compatible and therefore the tests are not failing due to incompatibility of underlaying testing platform. In addition, browser manager is able to download webdrivers and browsers binaries (CfT).

### CODE REVIEW

**Comment by quigamdev** [comment]
NOTE: Firefox does support this feature, but it doesn't work on GitHub Actions (reason unknown).

File: src/Tests/Riganti.Selenium.Core.Samples.Tests/BrowsersStabilityUiTests.cs [hunk 1]
```diff
@@ -70,15 +70,23 @@ public void SelectMethod_XPathToRoot_ExpectedException()
...
-                var elem = browser.Single("#span");
-                elem.Single("///***-*///@@##@šš+š++++---><<>''", By.XPath);
-            });
+                this.RunInAllBrowsers(browser =>
+                {
+                    browser.NavigateToUrl("/test/ClickTest");
+                    var elem = browser.Single("#span");
+                    elem.Single("///***-*///@@##@šš+š++++---><<>''", By.XPath);
+                });
+                throw new System.Exception("SelectMethod_InvalidXPathSelector_ExpectedException was supposed to fail!!!");
```
Comment (exyi) [inline src/Tests/Riganti.Selenium.Core.Samples.Tests/BrowsersStabilityUiTests.cs]: This does not really make sense to me :sweat_smile: , did you intend to use Assert.Throws?

**Review by exyi — APPROVED** [review]
:+1:

File: src/Tests/Riganti.Selenium.Core.Samples.Tests/BrowsersStabilityUiTests.cs (hunk 1 above)
Comment (quigamdev) [reply src/Tests/Riganti.Selenium.Core.Samples.Tests/BrowsersStabilityUiTests.cs]: Assert.Throw<> checks for specific exception. The mistake is incorrect place of the `throw new exc...`. I have corrected it.

**Pull request merged**